pandas/tests/resample/test_datetime_index.py

```diff
@@ -1286,7 +1291,7 @@ def test_resample_consistency():
     tm.assert_series_equal(s10_2, rl)
 
 
-dates1 = [
+dates1: List[Union[datetime, NaTType]] = [
```

Comment:
ahh this, should be a first class type (the Union)

Reply:
create a type in next commit